Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptEditController.java

```diff
@@ -216,13 +206,79 @@ public String confirmlEdit(@PathVariable("id") String id,
 				}
 			}
 		}
-
 		ConceptEntry[] arraySynonyms = new ConceptEntry[synonyms.size()];
+
 		int i = 0;
+		StringBuffer jsonStringBuilder = new StringBuffer("{");

```

Comment:
Yes thats true. I had it as a separate method because in future if someone wants to add few attributes to JSON and also these synonym attributes it would be easier to call this method after adding the required attributes. 
If you want me to add all these code snippet into one private method I can do that.